packages/api/src/libs/queries/emp.ts

```diff
@@ -113,14 +113,27 @@ export default (appState: Dependencies) => {
       tvm,
       type: "emp",
     };
-    let gcr = "0";
+    // raw gcr is just the collateral divided by tokens outstanding
+    let gcrRaw = "0";
     try {
-      gcr = calcGcr(state).toString();
+      gcrRaw = calcGcr(state).toString();
+    } catch (err) {
+      // nothing
+    }
+    // gcr now also takes into consideration the identifier price, defaults to raw if anything happens
+    let gcr = gcrRaw;
+    try {
+      gcr = utils
+        .parseUnits(gcrRaw)
+        // if theres no identifier price, just divide by 1
```

Comment:
why do we assume 1? If we don't have a price, shouldn't we just return null or undefined or something?

Reply:
its dividing out 1 eth so the result ends up being the same as the raw gcr. i figured its probably best to fall back to displaying the raw gcr rather than nothing if theres no identifier price available. what do you think?

Reply:
Hmmm, okay. We should probably align on some error handling/data availability protocol for all fields so the FE has a way to detect if the value isn't available.

Intuitively, subbing in the raw GCR seems like a decision we'd want to make on the FE instead of on the backend.